hub/admin/extend_user.py

```diff
@@ -310,3 +310,14 @@ def _get_message(self, singular: bool, grace_period: int) -> str:
             message += f'View <a href="{url}">trash.</a>'
 
         return mark_safe(message)
+
+    def save_model(self, request, obj, form, change):
+        if obj.is_superuser:
+            if config.SUPERUSER_AUTH_ENFORCEMENT:
```

Comment:
No need to nest

`if obj.is_superuser and config.SUPERUSER_AUTH_ENFORCEMENT and ...`